CHANGELOG.md

```diff
@@ -24,12 +24,12 @@ changes.
     and verify signature of received messages is from a known party
     and of course valid.
 
-- **BREAKING**Protocol change: Wait for all transactions requested in
-  a snapshot to be seen before acknowledging it, and only send
-  transaction ids in snapshot requests
+- **BREAKING**Protocol change: Wait for all transactions requested in a snapshot
```

Comment:
Why is this change here? It seems we might have different formatting for markdown, perhaps something to settle on somewhere?

Reply:
Yeah, I think my editor is using a formatter automatically